### PR TITLE
aspects/Spelling: Add coalaCorrect

### PR DESCRIPTION
**Pull request by Asnelchristian**
Please propose better names for this aspect.

Closes https://gitlab.com/coala/GSoC-2017/issues/236

### CODE REVIEW

**Comment by userzimmermann** [comment]
@Asnelchristian Why is just `coala` no enough in your opinion btw?

**Comment by userzimmermann** [comment]
@Asnelchristian Or what do you think about `coalaOnly`? With some description like that there is **only** one way to write `coala` :)

File: coalib/bearlib/aspects/Spelling.py [hunk 1]
```diff
@@ -33,3 +33,27 @@ class docs:
         ``aspects`` or ``aspect`` are always written with all lower-case
         letters or the term ``aspectsYEAH`` must match the exact expression.
         """
+
+
+@Spelling.subaspect
+class coalaCorrect:
+    """
+    This aspect dictates that the term ``coala`` must always be written with
+    with a lowercase `c`.
```
Comment (userzimmermann) [inline coalib/bearlib/aspects/Spelling.py]: Not only with lowercase `c`. It must be all lowercase :)

Comment (userzimmermann) [inline coalib/bearlib/aspects/Spelling.py]: @Asnelchristian @sils Hmm... Seems like it must not be **all** lowercase... ok :)

**Comment by userzimmermann** [comment]
ack https://github.com/coala/coala/pull/4328/commits/b0302a9c79fd7b07f6e556e91c09aacde41c64e7

**Comment by userzimmermann** [comment]
@rultor merge

**Comment by Asnelchristian** [comment]
ack https://github.com/coala/coala/pull/4328/commits/741d0929628a7bff62be0fd07f95edd343a4a458

**Comment by Asnelchristian** [comment]
@rultor merge

**Pull request merged**